### ISSUE TITLE
DatePicker: yearNavigator=select does not have proper keyboard support

### ISSUE THREAD

**Issue by nichirsch12** [primefaces/primefaces#14346]
### Describe the bug

A p:datePicker initialized with yearNavigator="select" creates the Year selector as a dropdown. However, this dropdown cannot be opened with space or enter like the monthNavigator can.

### Reproducer

Bug can be reproduced in PrimeFaces showcase, Navigator (Year Select):
https://showcase.primefaces.org/ui/input/datepicker/datePickerJava8.xhtml?jfwid=ac553

### Expected behavior

Year navigator dropdown should behave like the Month dropdown

### PrimeFaces edition

None

### PrimeFaces version

15.0.10

### JSF implementation

Mojarra

### JSF version

4.0

### Java version

17

**Comment by melloware** [comment]
Thanks fixed for 15.0.10 release

**Comment by nichirsch12** [comment]
Thank you!
